docker/README.md

```diff
@@ -0,0 +1,61 @@
+# VRX Docker support
+
+# Overview
+
+The directory contains a series of Dockerfiles designed to serve different setups
+for VRX. The usage is mainly controlled by `docker-compose` although docker files
+can be used independently if desired:
+
+ * `Dockerfile.base`: built on top of osrf desktop-full. Install
+   gz-harmonic from packages.osrfoundation.org repostiory and
+   the VRX dependencies using rosdep.
+
+   * `Dockerfile.builder`: built on top of `vrx-base`. Compiles the vrx packages
+     using colcon.
+
+   * `Dockerfile.devel`: built on top of `vrx-base`. development image with extra ROS 2 development packages
+     and other dev utilities. It configures the GPU support and a development
+     user. Runs a shell into the container with local sources mapped in /ws.
+
+# Development container
+
+The system is prepared to leave a running container with the
+VRX setup ready and the local clone of this repository mapped
+into the container in /ws (using docker volumes).
+
+The docker compose will perform:
+
+ * Build the necessary docker images
+ * Prepare the NVIDIA setup to run the container with GPU support
+ * Use `ubuntu` user for development
+
+A typical working run can consist on:
+
+```bash
+$ cd vrx/docker
+$ docker-compose run --rm devel
+...
+... 
+# docker building and return a shell inside the container
+$ ubuntu@locashot:/ws$ export ROS_DISTRO=jazzy
```

Comment:
Does it make sense to have this command exported, the VRX dependencies installed, and sourcing the `setup.bash` from ROS already configured?